### DISCUSSION TITLE
Downloading deleted Youtube video

### DISCUSSION THREAD

**Discussion by ghost** [community/community#140145] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I can't find specific instructions on how to download a deleted Youtube video on Github. I've looked for instructions to do this using yt-dlp. I don;t have file for the video, only the link. I got the link from the Wayback Machine, where the video has been added but not archived. Here's the title and link. 

Jack Frost ❄ Safe and Sound [Rise of the Guardians] Music Video
https://www.youtube.com/watch?v=7sYHt5vUHLI

**Comment by LiteBrite82** [comment]
Hi @Anandajoy ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by ghost** [reply]
Can yt-dlp be used to retrieve a video that was made private before it was deleted? Will the video be watchable?

**Comment by rafinhaleal871-lgtm** [comment]
can you reupload [Derpy Hooves Has A Sparta Remix by SonicAdventure2Freak
https://www.youtube.com/watch?v=Q_p7S-Ft7rQ
